src/python/bot/tasks/corpus_pruning_task.py

```diff
@@ -840,6 +875,12 @@ def _try_save_coverage_information():
         'Failed to save corpus pruning result: %s.' % repr(e))
 
 
+def choose_cross_pollination_strategy():
+  """Chooses cross pollination strategy. In seperate function to mock for
+    predictable test behaviror."""
+  return random.choice(['random', 'tagged'])
```

Comment:
as we use these string literals (random and tagged) more than once, it might be better to put them in some global variables to make sure they're defined at a single place and use those variables (kinda constants, but not really in python) in all other places